src/main/java/seedu/thanepark/model/ride/Ride.java

```diff
@@ -68,6 +68,10 @@ public Maintenance getDaysSinceMaintenance() {
         return daysSinceMaintenance;
     }
 
+    public Maintenance resetMaintenance() {
+        return new Maintenance("0");
```

Comment:
Is this supposed to be `daysSinceMaintenance = new Maintenance("0");`
Also, might want to make `Maintenance` immutable. Currently we can `setValue()`.